supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -83,6 +83,19 @@ For more information about Intel® trademarks, see the http://www.intel.com/cont
 
 *See also*:
 
+
+[discrete]
+[[as-expected]]
+==== image:images/no.png[no] alright (adjective)
+*Description*: "Expectations are relative, prefer using "correctly" instead."
```

Comment:
* Can you remove the quotation marks around the entire description?
* Maybe add a semicolon, and just say "use":
```
Expectations are relative; use "correctly" instead.
```